backend/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apk update && \
       postgresql-dev \
       imagemagick \
       geos-dev \
+      yarn \
```

Comment:
so we are going for yarn? If I remember correctly, Rails 7 adds new support for impartmaps, which should be simplification, but I don't mind yarn stuff

Reply:
Feel free to refactor this at any moment. I would love to see it with importmaps. I did not have time to check how to configure it with and remove all those packages from package.json. Just doing, the way I know it works at this moment, to do it quickly.